### DISCUSSION TITLE
Brightness Adjust concept

### DISCUSSION THREAD

**Discussion by meteorquake** [sumatrapdfreader/sumatrapdf#3042] [Ideas]
In these days of looking after eye strain, can I suggest Sumatra have a brightness control.
Specifically, a lot of documents are black writing on white background, often scans rather than text, and what would be ideal would be to be able to mute the white by making it grey, so essentially a slider (or value box) that compresses the brightness range between 0 and the slider, and when brought down to 0 it all becomes black and then when taken into the negative the colours are reversed and the brightness increases, so effectively the background has become black and as the slider goes further into the negative the foreground progressively lightens through grey to white.
I'm aware you can launch in invert mode but this suggestion is finer-tuned, and also that computers also have night modes etc but it's often the case you want to selectively do this. So for example on your computer you may want photos you are working with to be full colour but the PDFs you are reading to be muted in this way...
David

**Comment by GitHubRulesOK** [comment]
thats why there is a range that you can alter once in a while or by shortcuts https://github.com/ysl2/my-notes/issues/1#issuecomment-968262399

Many examples of why users cant grasp it works for paper but not full colour images https://github.com/sumatrapdfreader/sumatrapdf/issues/79

**Comment by meteorquake** [comment]
Thanks - that's certainly an improvement on absence. It looks like it's a global preference, so presumably I'd need to have different sumatra settings files (-appdata command etc) and launch a PDF by one or the other according to whether it's a document with colour pictures or not...
I certainly appreciate the need for it to only apply to areas that look monochrome though obviously that's an ideal wish. :)
d

**Comment by GitHubRulesOK** [comment]
@meteorquake for flip between inverted modes use I key but for day/night you can
 **use two command line shortcuts fixing both color ranges** (or a second appdata)
 **shortcuts keep one history** appdata keeps 2 histories e.g.  day books and night books

**Comment by meteorquake** [comment]
Yes... I'll probably make different file extensions so files can open by default in the right one.
It would be good if the i key could cycle through settings so e.g. BackgroundColor = #ffffff,#808080,#000000 would represent a cycle of three options, the first one being the startup value.
d

**Comment by GitHubRulesOK** [comment]
@meteorquake 

![image](https://user-images.githubusercontent.com/12754913/197337087-7ad65dce-7b56-4911-b037-919ac6288aeb.png)

**Comment by meteorquake** [comment]
The above is only plaster to the issue though. I'm sure I'm not alone in having PDFs which are mostly black and white text but have photos scattered through them that need examining, so you need to be able to switch the display easily on a single file that is actively open. The easy solution (I think) is to have the i button cycle through some predetermined values (whose default could be the current definition if wished)...
David
